docs/source/using/managing-minishift.adoc

```diff
@@ -53,6 +53,42 @@ the GitHub issue https://github.com/minishift/minishift/issues/179[#179].
 The link:../command-ref/minishift_delete{outfilesuffix}[`minishift delete`] command deletes the OpenShift cluster,
 and also shuts down and deletes the Minishift VM. No data or state are preserved.
 
+[[choosing-iso-image]]
+== Choosing the ISO image
+
+When you start Minishift, it downloads a live ISO image that the hypervisor uses to
+provision the host VM.
+
+The following ISO images are available:
+
+- link:https://github.com/minishift/minishift-b2d-iso/releases[Boot2Docker] (Default). This ISO is
+based on the link:https://github.com/boot2docker/boot2docker[Boot2Docker] ISO, which is a
+lightweight Linux distribution optimized for containerized application development. This ISO image
+is optimized for development but not for production, and is much smaller in size from the CentOS ISO image.
+
+- link:https://github.com/minishift/minishift-centos-iso/releases[CentOS]. This ISO contains
+the link:https://www.centos.org/[CentOS] operating system, which is an enterprise-ready Linux
+distribution. This ISO image provides a production-ready environment.
```

Comment:
I think we should be saying something like:

CentOS operating system, which is an enterprise-ready Linux more mirroring a production environment for OpenShift.

> This ISO image provides a production-ready environment.

I don't think we can say this. Minishift does not provide a production environment. It should never be used in a production setting. We should not imply this. The image mirrors more a production environment for an actual OpenShift install.